Dockerfile

```diff
@@ -20,5 +20,11 @@ RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
 
 COPY . /code/
 
-# Heroku uses PORT, Azure App Services uses WEBSITES_PORT, Fly.io uses 8080 by default
-CMD ["sh", "-c", "uvicorn server.main:app --host 0.0.0.0 --port ${PORT:-${WEBSITES_PORT:-8080}}"]
+# Find placeholder domain and replace with actual domain
+ARG RENDER_EXTERNAL_HOSTNAME
+RUN grep -rl "your-app-url.com" . | xargs sed -i "s/your-app-url.com/${RENDER_EXTERNAL_HOSTNAME}/g"
+
+# The Blueprint file can inject the hostname into the environment, but source code expects http://hostname format
+ARG WEAVIATE_HOSTNAME
+# Render and Heroku use PORT, Azure App Services uses WEBSITES_PORT, Fly.io uses 8080 by default
+CMD ["sh", "-c", "env WEAVIATE_HOST=http://${WEAVIATE_HOSTNAME} uvicorn server.main:app --host 0.0.0.0 --port ${PORT:-${WEBSITES_PORT:-8080}}"]
```

Comment:
Could we set the ENV var in the Dockerfile instead of in the command itself?

Reply:
Yes, good call. Using an `ENV` command is cleaner.